src/wp-includes/rest-api/endpoints/class-wp-rest-users-controller.php

```diff
@@ -647,6 +647,23 @@ public function create_item( $request ) {
 			}
 		}
 
+		/**
+		 * Filters the type of notification sent upon new user registration.
+		 *
+		 * This filter allows customization of the notification type when a new user is created via REST API.
+		 *
+		 * @param string $notify  Determines who gets notified. Accepts:
+		 *                        - 'admin' (default) or an empty string: Notify only the site administrator.
+		 *                        - 'user': Notify only the new user.
+		 *                        - 'both': Notify both admin and user.
+		 *                        - 'false': Disable notifications entirely.
+		 * @param int    $user_id User ID.
+		 */
+		$notify = apply_filters( 'rest_wp_user_created_notification', $notify = 'admin', $user_id );
+		if ( $notify && 'false' != $notify ) {
```

Comment:
```suggestion
		if ( $notify && 'false' !== $notify ) {
```

Better to use strict comparison for a more robust implementation